"""
Errores personalizados
"""


class InputError(Exception):
    """
    Excepcion para errores de entrada
    """
    pass


class OutputError(Exception):
    """
    Excepcion para errores de salida
    """
    pass